src/20230817_AaveV3_Eth_SDAIOnboarding/AaveV3_Ethereum_SDAIOnboarding_20230817.t.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {GovHelpers} from 'lib/aave-helpers/src/GovHelpers.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'lib/aave-helpers/src/ProtocolV3TestBase.sol';
+import {AaveV3_Ethereum_SDAIOnboarding_20230817} from './AaveV3_Ethereum_SDAIOnboarding_20230817.sol';
+
+/**
+ * @dev Test for AaveV3_Ethereum_SDAIOnboarding_20230817
+ * command: make test-contract filter=AaveV3_Ethereum_SDAIOnboarding_20230817
+ */
+contract AaveV3_Ethereum_SDAIOnboarding_20230817_Test is ProtocolV3TestBase {
+  AaveV3_Ethereum_SDAIOnboarding_20230817 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 17935112);
+    proposal = new AaveV3_Ethereum_SDAIOnboarding_20230817();
+  }
+
+  function testProposalExecution() public {
+    ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot(
+      'preAaveV3_Ethereum_SDAIOnboarding_20230817',
+      AaveV3Ethereum.POOL
+    );
+
+    GovHelpers.executePayload(vm, address(proposal), AaveGovernanceV2.SHORT_EXECUTOR);
+
+    ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot(
+      'postAaveV3_Ethereum_SDAIOnboarding_20230817',
+      AaveV3Ethereum.POOL
+    );
+
```

Comment:
would also put e2e test for sDAI (using underlying address as its not added yet on address-book)

Reply:
Done